### DISCUSSION TITLE
Show sticky notes

### DISCUSSION THREAD

**Discussion by ghost** [ahrm/sioyek#601] [Q&A]
I did not understand how to show sticky notes, for examples the ones present [here](https://web.stanford.edu/~mrosenfe/soc_meth_proj3/matrix_OLS_NYU_notes.pdf)

![image](https://user-images.githubusercontent.com/118444583/212606228-de740a5b-6b74-4ab4-be98-2bc222adb879.png)

**Comment by ahrm** [comment]
You can't.

**Reply by ghost** [reply]
That's bad. Can we have that feature?

**Reply by ahrm** [reply]
~~I think muPDF (the engine we use to render PDFs) doesn't have this feature. So you should contact mupdf devs.~~

**Reply by ahrm** [reply]
Oh wait that sticky note is not part of the PDF, it is just how to devs chose to show it. You can import the notes as sioyek bookmarks using the import bookmark extension.

**Reply by ghost** [reply]
I was looking for sticky notes too, and now I am using Document Viewer 1.0.0, from AUR repository, which have ones, instead of Sioyek, although I would like to use Sioyek with its amazing-in-all-other-aspects features.
